chain-core/src/common/fixed.rs

```diff
@@ -25,45 +26,196 @@ fn continued_fraction(
 }
 
 // https://en.wikipedia.org/wiki/Exponential_function#Continued_fractions_for_ex
-fn exp_series(
+fn exp2_series(
     x: FixedNumber,
+    y: FixedNumber,
 ) -> (
     impl Iterator<Item = FixedNumber>,
     impl Iterator<Item = FixedNumber>,
```

Comment:
as this return type is a bit long, I think it'll be good to at least give it a type alias with some comment how it's meant to be used, e.g.:
```
type FractionSeries = (impl Iterator<Item = FixedNumber>, impl Iterator<Item = FixedNumber>);
```

And my guess is that the usage / implementation is a bit different than the RosettaCode with macros -- so will be good to capture / documentation that (I guess one first needs to remove the head of the first iterator, and then zip it with the second iterator?)

Reply:
It will complains:
```
rust says `impl Trait` not allowed outside of function and inherent method return types
```

If I only make an alias for `impl Iterator<Item = FixedNumber>`, it still complains:
```
`impl Trait` in type aliases is unstable
note: for more information, see https://github.com/rust-lang/rust/issues/63063
```

Reply:
Something like `struct FractionSeries<T: impl Iterator<Item = FixedNumber>>(pub T, pub T)` would complain about trait objects?

Reply:
if it's wrapped in a struct, I guess one other alternative would be:

```
impl FractionSeries ... {
   pub fn continued_fraction(self, iteration_count: usize) -> FixedNumber {
      ... continued_fraction code...
  }

} 
```

Reply:
we can come back to that in a different PR